dir.props

```diff
@@ -72,7 +72,7 @@
 
   <ItemGroup>
```

Comment:
Not necessary for this PR, but I learned our dir.props should not include the line:
```
  <DnuSourceList Include="https:%2F%2Fwww.myget.org/F/dotnet-coreclr/" />
```
This appears to be one of the reasons for build failure as we were moving to the new enterprise myget account (I think it wasn't forwarding properly when I tried to build). It is likely to cause us more problems in the future.  @iamjasonp -- do you think it makes sense to include that change in this PR?